.github/actions/stage-release-artefacts/tests/test_stage.py

```
@@ -0,0 +1,371 @@
+            normalize_windows_paths=True,
+        )
+
+        contents = output_file.read_text(encoding="utf-8")
+        assert "path=C:/Users/test" in contents
+
+
+class TestStageArtefacts:
+    """Tests for the stage_artefacts function."""
+
+    def test_stages_artefact(
+        self, tmp_path: PathType, monkeypatch: pytest.MonkeyPatch
```

Ruff (0.14.10)

Unused method argument: `monkeypatch`

(ARG002)